messages/reminders.py

```diff
@@ -10,7 +10,8 @@
 "Did the previous week play out close to how you planned it? React with either a thumbs up or thumbs down.",
 "Your poor planning should not become someone elses emergency. Please take a few minutes to plan your week and schedule all the events that you might need!",
 "Please pay attention to mailers from 1to1help even if their material is of no use to you right now. Do not wait until you need the help. Gradually accumulating byte sized tips will most likely prove crucial in the future to you and people you care about.",
-"As you plan your week, please factor in time to remain minimally healthy. Schedule frequent breaks to stand up, stretch and walk multiple times during the day. Make an effort to apply the tips sent by 1to1help.net"]
+"As you plan your week, please factor in time to remain minimally healthy. Schedule frequent breaks to stand up, stretch and walk multiple times during the day. Make an effort to apply the tips sent by 1to1help.net",
+"Avoiding burnout is important for an early employee. Make sure to pace yourself well. Finish your alloted vacation days every year. Take frequent breaks during the day to stand up, stretch and walk. Recognize when you are getting bored or overwhelmed and speak up. Accept the fact that it is OK to fail when you genuinely try. Ask your 1:1 manager for more tips."]
```

Comment:
We are using ',' in end of each statement. But here we are using '.'

Reply:
@Annapooraniqxf2 - the comma is after each list element. The final list element does not need a comma.